source/reference/minio-mc-admin/mc-admin-trace.rst

```diff
@@ -270,14 +273,19 @@ Syntax
         
           The maximum Time To First Byte for the Call.
 
+      * - Avg Size
+        - Average size of calls.
```

Comment:
Maybe out of scope for this PR, but do we want to continue to capitalize `Call`? Seeing as nuking `Tenant` is on my own to-do list.

Reply:
I can do a cull.

Reply:
Candidly I'm not sure what "call" means in this operation.

I think what we mean here is "operation" and maybe "response"

e.g.

The maximum Time to First Byte in response to a client operation or call

is wordy as all get out, but I think more precise?

Nitty though

Reply:
Applied your wording suggestion throughout.